README.Rmd

```diff
@@ -92,6 +97,8 @@ arc_select(
 )
 ```
 
+### Creating a `SpatRaster` from an ArcGIS ImageServer
+
 A `SpatRaster` object from the `{terra}` package can be extracted from an `ImageServer` using `arc_raster()`.
 
 `arc_raster()` requires a `bbox` class object from the `sf` package to define the area to be extracted. You can optionally specify the `width` and `height` of the resultant image. Use `format` to define what type of image is returned.
```

Comment:
```suggestion
`arc_raster()` will extract the area defined by `xmin`, `ymin`, `xmax`, and `ymax`. You can optionally specify the `width` and `height` of the resultant image. Use `format` to define what type of image is returned.
```

sorry... just catching my own issues here...

Reply:
A bit unrelated by why does arc_raster not provide a bbox argument? Feels a little fiddly to require users to define each of those parameters separately when a bbox is basically just a names list of those parameters.

Reply:
The endpoint that is used `export` does not permit adding any spatial filters (at least that I can tell). I also initially had it so that a `bbox` object could be used. But then I found it somewhat odd that you would have to create a bbox with a specified CRS from sf to get a terra object. I felt it made more sense (and consistent with other tooling in & out of the R ecosystem) to provide the x/y min/max values 

export-image docs:  https://developers.arcgis.com/rest/services-reference/enterprise/export-image.htm